java-frontend/src/test/files/se/SeEngineTest.java

```diff
@@ -109,3 +109,38 @@ boolean equals(Object obj) {
   }
 
 }
+
+class DefaultValues {
+
+  void localVars() {
+    boolean a; // this is basically noop
```

Comment:
I tend to disagree, this is not a noop : for me noop implies there is an operation while there is none here. It is a declaration. If I'm correct what you meant to say is more about the fact that this line has no impact on constraint on value. Really going for detail here I admit but I would drop that comment : useless and confusing.